docs/services/gcp-services-guide.md

```diff
@@ -0,0 +1,536 @@
+    Status:                False
+    Type:                  Ready
+    Last Transition Time:  2019-09-16T13:46:42Z
+    Reason:                Successfully reconciled managed resource
+    Status:                True
+    Type:                  Synced
+Events:                    <none>
+```
+
+*Note: You must wait until the claim becomes bound before continuing with this
+guide. It could take a few minutes for GCP to complete CloudSQL creation.*
+
+#### Connecting MySQL Instance and GKE Cluster
+Before we install Wordpress, we need establish connectivity between our MySQL
+database and our GKE cluster. We can do this by creating a [Private Service 
+Connection](https://cloud.google.com/vpc/docs/configure-private-services-access).
+
+You can create it following the link above or use Crossplane to do it:
+
+* Create a `GlobalAddress` and `Connection` resources:
+```bash
+cat > network.yaml <<EOF
+---
+# example-globaladdress defines the IP range that will be allocated for cloud services connecting
+# to the instances in the given Network.
+apiVersion: compute.gcp.crossplane.io/v1alpha2
+kind: GlobalAddress
+metadata:
+  name: example-globaladdress
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  name: example-globaladdress
+  purpose: VPC_PEERING
+  addressType: INTERNAL
+  prefixLength: 16
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+---
+# example-connection is what allows cloud services to use the allocated GlobalAddress for communication. Behind
+# the scenes, it creates a VPC peering to the network that those service instances actually live.
+apiVersion: servicenetworking.gcp.crossplane.io/v1alpha2
+kind: Connection
+metadata:
+  name: example-connection
+  namespace: $INFRA_NAMESPACE
+spec:
+  providerRef:
+    name: gcp-provider
+    namespace: $INFRA_NAMESPACE
+  reclaimPolicy: Delete
+  parent: services/servicenetworking.googleapis.com
+  network: projects/$PROJECT_ID/global/networks/$NETWORK_NAME
+  reservedPeeringRanges:
+    - example-globaladdress
+EOF
+
+kubectl apply -f network.yaml
+```
+
+* You can verify creation with the following command and output:
+
+*Command*
+```bash
+kubectl -n $INFRA_NAMESPACE get connection example-connection -o custom-columns='NAME:.metadata.name,FIRST_CONDITION:.status.conditions[0].status,SECOND_CONDITION:.status.conditions[1].status'
+```
+*Output*
+```
+NAME                 FIRST_CONDITION   SECOND_CONDITION
+example-connection   True              True
+```
+Wait for both conditions to be true to continue.
+
+## Install Wordpress
+Installing Wordpress requires creating a Kubernetes `Deployment` and load
+balancer `Service`. We will point the deployment to the `wordpressmysql` secret
+that we specified in our claim above for the Wordpress container environment
+variables. It should have been populated with our MySQL connection details after
+the claim became `Bound`.
```

Comment:
As a user, is there a document I can read to learn more about what this is talking about? Could I get a link to it from this document?

Reply:
I am not sure whether there is a doc that talks about the binding status except for @hasheddan 's podcast. I'll add more context there.